### PR TITLE
feat: resize image helper 

### PR DESCRIPTION
**Pull request by dimatretyak**
This PR resolves []

### Description
This PR adds the ability to resize/scale image 

### PR Checklist

- [x] I tested my changes on **iOS** / **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- Add image resizer - dimatretyak

</details>

Need help with something? Have a look on our [docs], or get in touch with us.

[app state migration]: /docs/adding_state_migrations.md
[feature flag]: /docs/developing_a_feature.md
[docs]: /docs/README.md

### CODE REVIEW

File: Podfile.lock [hunk 1]
```diff
@@ -28,19 +28,19 @@ PODS:
     - AppAuth/ExternalUserAgent (= 1.4.0)
   - AppAuth/Core (1.4.0)
   - AppAuth/ExternalUserAgent (1.4.0)
-  - Appboy-iOS-SDK (4.4.3):
-    - Appboy-iOS-SDK/UI (= 4.4.3)
-  - Appboy-iOS-SDK/ContentCards (4.4.3):
+  - Appboy-iOS-SDK (4.4.2):
```
Comment (gkartalis) [inline Podfile.lock:31]: 🤔  Weird that `Appboy-iOS-SDK` was downgraded vs all of the rest that got upgraded

File: src/app/utils/resizeImage.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  onlyScaleDown?: boolean
+  mode?: ResizeMode
+}
+
+export const resizeImage = (options: ResizeImageOptions) => {
+  const {
+    uri,
+    width,
+    height,
+    outputformat = "JPEG",
```
Comment (gkartalis) [inline src/app/utils/resizeImage.ts:21]: **question:** I read in the docs of [react-native-image-resizer](https://github.com/bamlab/react-native-image-resizer#api) under compressFormat it was saying that it is only supported in Android. 

In iOS it defaults to HEIC? And do we support `HEIC` on searching images in tineye?

**Review by gkartalis — DISMISSED** [review]
Looking good! Just left one question and one comment

File: src/app/utils/resizeImage.ts (hunk 2 above)
Comment (dimatretyak) [reply src/app/utils/resizeImage.ts:21]: > I read in the docs of [react-native-image-resizer](https://github.com/bamlab/react-native-image-resizer#api) under compressFormat it was saying that it is only supported in Android.

After checking, it turned out that this also works for iOS 😅

Comment (dimatretyak) [reply src/app/utils/resizeImage.ts:21]: > In iOS it defaults to HEIC? And do we support HEIC on searching images in tineye?

good question. I'll check it

File: Podfile.lock (hunk 1 above)
Comment (dimatretyak) [reply Podfile.lock:31]: this is some kind of problem on my part, I'm dealing with it now

Comment (gkartalis) [reply Podfile.lock:31]: let me know if you need anything 🙃

File: src/app/utils/resizeImage.ts (hunk 2 above)
Comment (dimatretyak) [reply src/app/utils/resizeImage.ts:21]: TinEye supports only following formats JPEG, PNG, WEBP, GIF, BMP, and TIFF

File: Podfile.lock (hunk 1 above)
Comment (dimatretyak) [reply Podfile.lock]: Fixed

File: src/app/utils/resizeImage.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    quality = 100,
+    rotation = 0,
+    outputPath,
+    keepMeta,
+    onlyScaleDown,
```
Comment (gkartalis) [inline src/app/utils/resizeImage.ts:26]: **question(non-blocking):** should we default that to true to make sure that in no case we are not scaling the size up and we always scale the images down?

I leave it up to you to decide 👍

**Review by gkartalis — APPROVED** [review]
🚀

File: src/app/utils/resizeImage.ts (hunk 3 above)
Comment (dimatretyak) [reply src/app/utils/resizeImage.ts:26]: if you don't mind, I'll leave the default behavior. I think this way there will be less confusion in situations where the developer wanted to change the size of the image, but in the end it remained the same

Comment (gkartalis) [reply src/app/utils/resizeImage.ts:26]: Sounds good to me 👍

**Pull request merged**